src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -25,10 +25,26 @@ function logCapturedError(capturedError : CapturedError) : void {
       willRetry,
     } = capturedError;
 
+    const {
+      message,
+      name,
+      stack,
+    } = error;
+
+    const errorSummary = message
+      ? `${name}: ${message}`
+      : name;
+
     const componentNameMessage = componentName
       ? `React caught an error thrown by ${componentName}.`
       : 'React caught an error thrown by one of your components.';
 
+    // Error stack varies by browser; normalize it for our logging.
+    const formattedCallStack = stack.replace(new RegExp(`^${errorSummary}\n`), '')
```

Comment:
This would need to be escaped in case it includes regex metachars. How about just:

```
if (stack.slice(0, errorSummary.length) === errorSummary) {
  stack = stack.slice(errorSummary.length);
}
if (stack.charAt(0) === '\n') {
  stack = stack.slice(1);
}
```